tests/test_groups_api.py

```diff
@@ -24,3 +24,17 @@ def test_create_group_api(api_client, authenticate_user_with_token):
     api_client.credentials()
 
     assert response.status_code == status.HTTP_201_CREATED
+
+
+@pytest.mark.django_db
+def test_get_groups_api(api_client, authenticate_user_with_token):
```

Comment:
I would rather the name suggests the functionality being tested as opposed to saying 'api' or the http method used. for example test_group_list